portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.utils.CommentDataStore;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles deleting comment content. */
+@WebServlet("/delete-comment")
+public class DeleteCommentServlet extends HttpServlet {
+
+    private CommentDataStore commentDataStore;
+
+    private static final String KIND = "Comment";
+    // TODO: actually this constant is used in both Comment and DeleteCommentServlet,
+    //  seems not proper to put it private static final, how i shall do it?
+
+    @Override
+    public void init() {
+        this.commentDataStore = new CommentDataStore(DatastoreServiceFactory.getDatastoreService());
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        long id = Long.parseLong(request.getParameter("id"));
+
+        this.commentDataStore.delete(KIND, id);
```

Comment:
Add TODO for error handling and param validation. 
There are a lot of errors that can happen here.
Id wrong format
Id does not match any comment
Id of comment not belong to you

Reply:
Fixed